meta-client/src/main/java/org/apache/helix/metaclient/api/MetaClientInterface.java

```diff
@@ -152,6 +152,15 @@ public Stat (EntryMode mode, int version, long ctime, long mtime, long etime) {
    */
   void create(final String key, final T data, final EntryMode mode);
 
+  /**
+   * Create an entry of given EntryMode with given key and data. If any parent node in the node
+   * hierarchy does not exist, then the parent node will attempt to be created. The entry will not be created if
+   * there is an existing entry with the same full key.
+   */
+  void createFullPath(final String key, final T Data, final EntryMode mode);
```

Comment:
Since we have `deleteRecursively`, how about `createRecursively` ?

Reply:
Updated naming to match